app/assets/javascripts/modules/tabs.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+moj.Modules.tabs = {
+  listSelector: 'ul.tabs',
+  panelSelector: 'section.tab-panel',
+  activeClass: 'active',
+
+  init: function() {
+    var self = this;
+
+    self.$list = $(self.listSelector);
+
+    if(self.$list.length) {
+      self.$tabs = self.$list.find('li');
+      self.$panels = $(self.panelSelector);
+
+      self.storeSlugs();
+      self.showInitialTab();
+      self.bindEvents();
+    }
+  },
+
+  bindEvents: function() {
+    var self = this;
+
+    self.$list.on('click', function(e) {
+      self.selectTab($(e.target));
+    });
+  },
+
+  showInitialTab: function() {
+    var self = this,
+      docHash = document.location.hash,
+      tabIndex = 0;
+
+    if(docHash.length) {
+      var checkSlug = docHash.slice(1);
+      var slugIndex = self.slugs.indexOf(checkSlug);
+      if (slugIndex > 0) {
+        tabIndex = slugIndex;
+      }
+    }
+
+    self.showTab(tabIndex);
+  },
+
+  storeSlugs: function() {
+    var self = this;
+
+    self.slugs = [];
```

Comment:
Don't know what's your preference but you may replace this loop with ES6's `map()`:

```javascript
  storeSlugs: function() {
    this.slugs = this.$tabs.map((_, tab) => {
      encodeURIComponent($(tab).text());
    });
  },
```

Reply:
This is client-side and we're not transpiling atmo. If `map()` works in ES5, then fine. 😄

Reply:
It's been a long day...

Reply:
👍 🛌